packages/code-du-travail-frontend/src/layout/Footer.js

```diff
@@ -83,6 +83,11 @@ const Footer = () => {
                       <StyledLink>Mentions légales</StyledLink>
                     </Link>
                   </StyledListItem>
+                  <StyledListItem>
+                    <Link passHref href="/accessibility">
```

Comment:
```suggestion
                    <Link passHref href="/accessibilité">
```

Reply:
Are we sure is it a good idea to put an accented character in the URL (and in the naming of a file)? Perhaps `/accessibilite`?

Reply:
ok for accessibilite